config/routes.rb

```diff
@@ -9,6 +9,7 @@
     member do
       get 'checksum'
       get 'file', format: false # no need to add extension to url
+      post 'content_diff', via: :post
```

Comment:
I don't think `via: :post` is necessary since this line already uses the `post` verb. I would expect to see the `via:` keyword arg for `match`-type routes.

Reply:
whoops!  thanks. will fix.